x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/engine/engine_indices.tsx

```diff
@@ -23,6 +23,7 @@ import { EnterpriseSearchEngineIndex } from '../../../../../common/types/engines
 
 import { CANCEL_BUTTON_LABEL } from '../../../shared/constants';
 import { generateEncodedPath } from '../../../shared/encode_path_params';
+import { indexHealthToHealthColor } from '../../../shared/indices/utils';
```

Comment:
@saarikabhasi has this work done in a separate PR: https://github.com/elastic/kibana/pull/149271

(also no reason to put that under indices specifically)

Reply:
that's slightly different from what I did here (function vs shared object). But I agree we should align.